src/etc/inc/plugins.inc.d/openvpn.inc

```diff
@@ -1004,6 +1004,18 @@ function openvpn_restart($mode, $settings)
 
     @unlink("/var/etc/openvpn/{$mode_id}.sock");
```

Comment:
maybe we only need the pid unlink for the pid file like here?

Reply:
Perhaps, but we need feedback to try and nail it down. Just covering all bases to start with.